packages/bento-frontend/src/pages/dashTemplate/sideBar/BentoCaseSearch.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Button } from '@material-ui/core';
+import {
+  resetUploadData, SearchBoxGenerator, SearchList,
+  UploadModalGenerator,
+} from '@bento-core/local-find';
+import store from '../../../store';
+import {
+  getAllSubjectIds, getAllIds,
+} from '../../dashboardTab/store/dashboardReducer';
+
+// Generate SearchBox Component
+const { SearchBox } = SearchBoxGenerator({
```

Comment:
@amattu2 Is it possible to move this file into local find package (@bento-core/local-find)?

Reply:
Sure. This file was moved to the Local Find package.